src/notation/view/widgets/editstyle.cpp

```diff
@@ -1088,6 +1148,51 @@ void EditStyle::setPage(int idx)
     }
 }
 
+//---------------------------------------------------------
+//   pageListRowChanged
+//---------------------------------------------------------
+
+void EditStyle::pageListRowChanged(int row)
+{
+    pageStack->setCurrentIndex(pageListMap[row]);
+}
+
+//---------------------------------------------------------
+//   pageListMoved
+//---------------------------------------------------------
+
+void EditStyle::pageListMoved(QModelIndex,int Start,int,QModelIndex,int End)
+{
+    if (End > Start) {
+        int startPageIndex = pageListMap[Start];
+        for (int i=Start; i < (End-1); i++) {
+            pageListMap[i] = pageListMap[i+1];
+        }
+        pageListMap[End-1] = startPageIndex;
+    } else {
+        int startPageIndex = pageListMap[Start];
+        for (int i=Start; i > End; i--) {
+            pageListMap[i] = pageListMap[i-1];
```

Comment:
here too I guess, for the -

Reply:
all fixed.

Reply:
Yep. looks OK code style wise now. But don't shoot me if the automated code style check find some more or even complains about some of the changes I requested ;-)

Reply:
you're doing a great job